classes/question_ui_renderer.php

```diff
@@ -438,18 +311,20 @@ private function set_input_values_and_readonly(DOMXPath $xpath, question_attempt
     /**
      * Removes remaining QuestionPy elements and attributes as well as comments and xmlns declarations.
      *
-     * @param DOMXPath $xpath
      * @return void
      */
-    private function clean_up(DOMXPath $xpath): void {
+    private function clean_up(): void {
         /** @var DOMNode|DOMNameSpaceNode $node */
-        foreach (iterator_to_array($xpath->query("//qpy:* | //@qpy:* | //comment() | //namespace::*")) as $node) {
+        foreach (iterator_to_array($this->xpath->query("//qpy:* | //@qpy:* | //comment() | //namespace::*")) as $node) {
             if ($node instanceof DOMAttr || $node instanceof DOMNameSpaceNode) {
                 $node->parentNode->removeAttributeNS($node->namespaceURI, $node->localName);
             } else {
                 $node->parentNode->removeChild($node);
             }
         }
+        /** @var DOMNode $root */
+        $root = $this->xpath->document->documentElement;
+        $root->removeAttributeNS(self::XHTML_NAMESPACE, "");
```

Comment:
Warum ist das nötig? (Antwort gern als Kommentar :wink:)

Und:
```suggestion
        /** @var DOMNode $root */
        $root = $this->xml->documentElement;
        $root->removeAttributeNS(self::XHTML_NAMESPACE, "");
```